preview/arcus-general.md

```diff
@@ -0,0 +1,71 @@
+# Arcus - Azure development in a breeze.
+
+## What is Arcus?
+A set of Codit-developed OSS libraries/tools that simplify building apps running on Microsoft Azure.
+It is Codit’s strategy to centralize company-wide best practices and use it across all our customers.
+
+* Built By Experts, For Experts
+  Sponsored by Codit! Maintained by a dedicated set of Codit experts that assures quality for our current and future clients.
+
+* Modular Design
+  All Arcus components are highly functional and composable. They can be used separately, but they are better together.
+
+* Open-Source & Free to use
+  The Arcus framework is constantly updated by contributions from Codit employees and abroad!
+  It is free to use under the MIT license.
+
+🚩 Are you doing the same thing at multiple customers? Then we are happy to hear! [Link to arcus GitHub]()
+
+## Why should you use Arcus?
+There exists many reasons to use Arcus in your current and future projects.
+
+* Time-saving
+  Arcus contains tremendous amounts of boilerplate code that helps you save loads of time during project development.
+  It helps with infrastructure like secure secret management, application-wide telemetry tracking, tedious DevOps operations. and even has fully development-ready project templates to get you started immediately.
+
+* Centralized
+  The entire Arcus framework is maintained and tested the same way, with the same set of dedicated experts. There exists a single point of truth were everything is maintained and made ready for your benefit.
+
+* Best practices
+  Every Arcus component has the company-wide best practices backed right into the code. It is secure by-default, tracks telemetry by itself, automates hard-to-setup DevOps tasks, and many more.
+
+## How to use Arcus?
+The Arcus components are available on NuGet, PowerShell Gallery and Python PyPi.
+
+Are you using Web API technology? Take a look at these components:
+* Arcus Web API project template
+* Arcus Azure Functions HTTP trigger
+* Arcus Web API
+
+Are you using messaging technology with Azure? Like Azure Service Bus? Take a look at these components:
+* Arcus Azure Service Bus Worker templates
+* Arcus Messaging
+
+Are you using repeated background work? Take a look at at these components:
+* Arcus Databricks Metrics project template
+* Arcus Background Jobs
+
+Are you using Azure resources in your DevOps pipeline? Take a look at these components:
+* Arcus Scripting
+
+Are you using ML technology? Take a look at these components:
+* Arcus ML
+* Arcus Azure ML
+
+For more information on all the different Arcus components: [separate page]().
+
+## Who's using Arcus?
```

Comment:
If I remember correctly there was a more extensive list in one of the Powerpoints from some time ago, I can't find it at the moment but it contained a list of customers and an amount of stars for each Arcus component they used.

Reply:
Ah, yeah, let's add that! 👍

Reply:
https://github.com/arcus-azure/arcus/pull/204/commits/47f67df39c795631893d67f15bd0823ae6dae3f2

Reply:
Now that I think about it, I'm not sure all these customers have agreed to be listed on the website. 
I did have agreement from Van Gelder to have them listed, but not sure about all the others.

Reply:
Ah, yeah, maybe the PP we got from Massimo is for internal use only?

Reply:
Yeah If I remember correctly that was the case. Maybe remove the customers you added in https://github.com/arcus-azure/arcus/commit/47f67df39c795631893d67f15bd0823ae6dae3f2 and check with Massimo if these customers can be listed or not?

Reply:
Send him a message a reverted it https://github.com/arcus-azure/arcus/pull/204/commits/da02b1358afedec24e30fa67ddbbf294b7f79e1f